config/prod/prow/config_knative.yaml

```diff
@@ -289,6 +289,10 @@ presubmits:
   - build-tests: true
   - unit-tests: true
   - integration-tests: true
+  knative-sandbox/kn-plugin-func:
+  - build-tests: true
+  - unit-tests: true
+  - integration-tests: true
```

Comment:
Could we start small, by disabling the Prow tests (func still has their tests running with GitHub Actions), and just enabling cla-bot and tide ? That way people already can work with the Prow flow while we are deciding how to continue with the CI setup.

Alternatively would could enable it and just add empty hook-scripts for the tests, which we will fill in later.

Reply:
We have been working pretty hard on our CI infrastructure for `func` and I would love it if we can continue using this while we work out the prow details. However that happens is fine with me, but if possible it would be great to not lose all of the work we've done on CI to date.